packages/client/src/pages/Login/AuthLogin.tsx

```diff
@@ -0,0 +1,111 @@
+import { MouseEvent, useState } from "react";
```

Comment:
Если объект MouseEvent используется только как тип, то лучше его отдельно импортировать через
```suggestion
import type { MouseEvent } from "react";
import { useState } from "react";
```
потому что import type на этапе сборки выпиливается из проекта